src/components/flame-graph/Canvas.js

```diff
@@ -58,6 +58,38 @@ class FlameGraphCanvas extends React.PureComponent<Props> {
     (this: any)._hitTest = this._hitTest.bind(this);
   }
 
+  getColor(implementation: Implementation, selfTimeRelative: number): string {
+    let h, s, l;
+    // The more self time a function has, the darker the color
+    // returned.  Do this by subtracting a lightness value from a base
+    // color for a particular implementation.  `a` and `b` are factors
+    // used in calcuating how much ligthness to subtract.  `limit` is
+    // the lower limit of the lightness.
+    let a, b, limit;
+
+    if (implementation === 'cpp') {
```

Comment:
I think here it would be nice to go through the list exhaustively using a switch statement. This way flow throws an error if the type ever changes:

```js
    switch (implementation) {
      case 'cpp':
        [h, s, l] = [358, 98, 82];
        a = 4;
        b = 5;
        limit = 55;
        break;
      case 'js':
        [h, s, l] = [188, 53, 71];
        a = 8;
        b = 6;
        limit = 40;
        break;
      case 'unknown':
        // Unknown implementation
        [h, s, l] = [77, 70, 71];
        a = 10;
        b = 10;
        limit = 40;
        break;
      default:
        throw new Error(
          `Unknown implementation case "${(implementation: empty)}".`
        );
    }
```